### DISCUSSION TITLE
How to hide PostgreSQL extension functions from database navigator?

### DISCUSSION THREAD

**Discussion by darko1979** [dbeaver/dbeaver#18875] [Q&A]
I have postgis extension enabled on one of my databases.
When I expand Functions branch it will display all my functions and postgis functions (there are 700+ functions in postgis).
It also shows extension types, views but there is only a few of them so it's not a problem as with functions.
Is there already an option to hide these from navigator?

**Comment by LonwoLonwo** [comment]
You can hide them only if they have special names - then you can use a filter on the folder and create the "exclude" list.

![2023-01-30 20_23_28-DBeaver Ultimate 22 3 3 - 11 24 324FFd](https://user-images.githubusercontent.com/45152336/215549243-cd6969bb-4d1e-49c9-8a44-f77afd9a8969.png)

**Comment by darko1979** [comment]
As I thought so. I know about the filters but that is not the solution I'm looking for.

**Comment by mkgrgis** [comment]
@darko1979, schema with PostgGIS functions isn't recommended for user tables and functions. It's more usefully to use other schemas with no PostgGIS functions for Your data. Also You can include this schemas to search_path for PostgreSQL.
